### DISCUSSION TITLE
Able to rencode once decoded

### DISCUSSION THREAD

**Discussion by infixremix** [Path-Check/paper-cred-demo#15] [Q&A]
I am doing this as a challenge rather than for personal use. It would compromise my morals. is there anyway i can rencode as ,ome is structured differently? Instead of using the UI. Just manually change the json ft.

**Comment by vitorpamplona** [comment]
Not without the private key.

**Reply by infixremix** [reply]
```
EU Digital Covid Certificate Verifiable QR
{
  "tag": 18,
  "value": [
    { "Map": [ [ 1, -7 ], [ 4, "osFRFyFIWdU=" ] ] },
    {},
    {
      "1": "GB",
      "4": 1642025280,
      "6": 1639433302,
      "-260": {
        "1": {
          "d": [
            {
              "ci": "URN:UVCI:01:GB:16394333023162LWFKGTX#7",
              "co": "GB-ENG",
              **"df": "2021-12-13T22:08:22+00:00",
              "du": "2022-01-12T22:08:00+00:00"**
              "is": "NHS Digital",
              "pm": 120,
              "po": [ "GB-ENG:1", "GB-ENG:2" ]
            }
          ],
          "dob": "1900-10-01",
          "nam": {
            "fn": "lastname",
            "gn": "firstname",
            "fnt": "LASTNAME",
            "gnt": "FIRSTNAME"
          },
          "ver": "1.0.0"
        }
      }
    },
    "9Oho/9EsW67074RaDLMg1qGbG+oDBiPmmK162DPpaJWf2EHS9ijd23g8RBlsCHyKySk/coU/w8ybcN9bnObAxw=="
  ]
}
```
jn the uk we use same infrastructure, however we use different initials like DF and DU. We just need to be able to change the figures is bold is there anyway i can do this? I have removed personal details. Our system is based offline it just uses system time and date and compares to the QR code,

**Reply by vitorpamplona** [reply]
The last part of your payload is a signature. In order to change anything in the top part, you need to change the signature as well. And changing the signature only works with the UK's private key.
